plaso/engine/artifact_filters.py

```diff
@@ -52,16 +58,28 @@ def __init__(self, artifacts_registry):
     self.file_system_find_specs = []
     self.registry_artifact_names = set()
     self.registry_find_specs = []
+    self.registry_find_specs_artifact_names = []
+    self.artifacts_trie = artifacts_trie.ArtifactsTrie()
 
   def _BuildFindSpecsFromArtifact(
-      self, definition, environment_variables, user_accounts):
+          self,
+          definition,
+          environment_variables,
+          user_accounts,
+          enable_artifacts_map=False,
+          original_registery_artifact_filter_names=None):
     """Builds find specifications from an artifact definition.
 
     Args:
       definition (artifacts.ArtifactDefinition): artifact definition.
       environment_variables (list[EnvironmentVariableArtifact]): environment
           variables.
       user_accounts (list[UserAccountArtifact]): user accounts.
+      enable_artifacts_map (Optional[bool]): True if the artifacts path map
+          should be generated. Defaults to False.
+      original_registery_artifact_filter_names (Optional[set[str]]): Set of
```

Comment:
@sa3eed3ed could change "registery" to "registry" in this PR.

Reply:
Done